### DISCUSSION TITLE
Uninstalling pnpm completely? (To reinstall with npm)

### DISCUSSION THREAD

**Discussion by Fuwn** [pnpm/pnpm#3282] [Q&A] [answered]
When originally installing pnpm, I used the Powershell method within Windows 10. I would like to reinstall it via NPM for the sake of organization.

How may I go about **fully** uninstalling pnpm?

**Comment by zkochan** [comment, marked as answer]
if you plan to reinstall it with npm, running `pnpm remove -g pnpm` should do it

**Reply by aalaap** [reply]
This command doesn't work for me with 7.0.0-rc.2:

```
 ERR_PNPM_NO_IMPORTER_MANIFEST_FOUND  No package.json (or package.yaml, or package.json5) was found in "C:\Users\John Doe\AppData\Local\pnpm\global-packages\5".
```

None of the commands listed on the uninstall page work. All produce the same error.

**Reply by zkochan** [reply]
track https://github.com/pnpm/pnpm/issues/4536

**Reply by deadcoder0904** [reply]
that didn't work as i got this error ` ERR_PNPM_CANNOT_REMOVE_MISSING_DEPS  Cannot remove 'pnpm': project has no 'dependencies'` but i removed it by cleaning up `C:\Users\<USER NAME>\AppData\Local\pnpm`.

**Reply by Rolanddoda** [reply]
Didn't work for me either: 

```
 ERR_PNPM_NO_IMPORTER_MANIFEST_FOUND  No package.json (or package.yaml, or package.json5) was found in "C:\Users\Roland Doda\AppData\Local\pnpm\global\5".

```

**Reply by pshergie** [reply]
I have this  `ERR_PNPM_CANNOT_REMOVE_MISSING_DEPS  Cannot remove 'pnpm': project has no 'dependencies'`

**Reply by Cristy94** [reply]
```
C:\Windows\System32>pnpm remove -g pnpm
 ERR_PNPM_NO_IMPORTER_MANIFEST_FOUND  No package.json (or package.yaml, or package.json5) was found in "C:\Users\XCS\AppData\Local\pnpm\global\5".
 ```
 
 Any fixes?

**Reply by Cristy94** [reply]
`npx pnpm install --global pnpm`

This fixed the error for me.
